bundles/org.openhab.binding.sagercaster/src/main/resources/ESH-INF/i18n/sagercaster_de.properties

```diff
@@ -0,0 +1,63 @@
+channel-type.sagercaster.forecast.state.option.G1 = Zunehmende oder sehr tr�be Bew�lkung, gefolgt von Niederschlag oder Schauern
+channel-type.sagercaster.forecast.state.option.G2 = Zunehmende Bew�lkung oder sehr bew�lkt, gefolgt von Niederschlag oder Schnee
+channel-type.sagercaster.forecast.state.option.H = Zunehmende Bew�lkung oder sehr bew�lkt, gefolgt von Niederschlag oder Schauern und Erw�rmung
+channel-type.sagercaster.forecast.state.option.J = Regeng�sse
+channel-type.sagercaster.forecast.state.option.K = Regeng�sse/Schnee und Erw�rmung
+channel-type.sagercaster.forecast.state.option.K1 = Regeng�sse und Erw�rmung
+channel-type.sagercaster.forecast.state.option.K2 = Schnee und Erw�rmung
+channel-type.sagercaster.forecast.state.option.L = Regeng�sse/Schnee und Abk�hlung
+channel-type.sagercaster.forecast.state.option.L1 = Regeng�sse und Abk�hlung
+channel-type.sagercaster.forecast.state.option.L2 = Regeng�sse und Abk�hlung
+channel-type.sagercaster.forecast.state.option.M = Niederschlag
+channel-type.sagercaster.forecast.state.option.N = Niederschlag und Erw�rmung
+channel-type.sagercaster.forecast.state.option.P = Niederschlag und Abk�hlung dann wahrscheinliche Besserung innerhalb von 24 Stunden
+channel-type.sagercaster.forecast.state.option.R = Niederschlag oder Schauer / Schnee und Besserung innerhalb von 12 Stunden
+channel-type.sagercaster.forecast.state.option.R1 = Niederschlag oder Schauer und Besserung innerhalb von 12 Stunden
+channel-type.sagercaster.forecast.state.option.R2 = Niederschlag oder Schnee und Besserung innerhalb von 12 Stunden
+channel-type.sagercaster.forecast.state.option.S = Niederschlag oder Schauer / Schnee und Verbesserung innerhalb von 12 Stunden und Abk�hlung
+channel-type.sagercaster.forecast.state.option.S1 = Niederschlag oder Regeng�sse und Besserung innerhalb von 12 Stunden und Erfrischung
+channel-type.sagercaster.forecast.state.option.S2 = Niederschlag oder Schnee und Verbesserung innerhalb von 12 Stunden und Abk�hlung
+channel-type.sagercaster.forecast.state.option.T = Niederschlag oder Schauer / Schnee und schnelle Besserung innerhalb von 6 Stunden
+channel-type.sagercaster.forecast.state.option.T1 = Niederschlag oder Schauer und schnelle Besserung innerhalb von 6 Stunden
+channel-type.sagercaster.forecast.state.option.T2 = Niederschlag oder Schnee und schnelle Besserung innerhalb von 6 Stunden
+channel-type.sagercaster.forecast.state.option.U = Niederschlag oder Schauer / Schnee und schnelle Besserung innerhalb von 6 Stunden, dann Abk�hlung
+channel-type.sagercaster.forecast.state.option.U1 = Niederschlag oder Schauer und schnelle Besserung innerhalb von 6 Stunden, dann Abk�hlung
+channel-type.sagercaster.forecast.state.option.U2 = Niederschlag oder Schnee und schnelle Besserung innerhalb von 6 Stunden, dann Abk�hlung
+channel-type.sagercaster.forecast.state.option.W = Niederschlag oder Schauer / Schnee, gefolgt von gutem Sonnenwetter innerhalb von 6 Stunden und Erfrischung
```

Comment:
We have three time option "W". Same in FR translations.

I would just say "Wetter" instead of "Sonnenwetter".